### PR TITLE
created badges controller

### LINKED ISSUE #18: As a repo owner I need a readme badge
## Goal
Provide a badge for repo README.md files

### PR DESCRIPTION
**Pull request by aaronmallen**
DN: this will need some refactoring later on.
resolves #18

### CODE REVIEW

**Comment by aaronmallen** [comment]
@samsheff don't merged this yet.

**Comment by aaronmallen** [comment]
@samsheff assuming it passes this is ready for QA now.

File: app/controllers/badges_controller.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  # TODO: hide behind authentication
+  def index
+    repo = Repo.find(params[:repo_id])
+    @badge = generate repo.latest_changes_count
+    render xml: @badge
+  end
+
+  private
+    def generate(count)
+      shield_uri = "http://img.shields.io/badge/changes-" + count.to_s + "-blue.svg"
```
Comment (samsheff) [inline app/controllers/badges_controller.rb]: @aaronmallen Maybe we can add support for different colors than blue programmatically? Assuming that's possible with their API, could be useful. Another idea would be to move this to a non-AR model, this way we can add support for more functionality without beefing up the controller too much.

**Comment by samsheff** [comment]
@aaronmallen Made a comment, otherwise this looks good.

File: app/controllers/badges_controller.rb (hunk 1 above)
Comment (aaronmallen) [inline app/controllers/badges_controller.rb]: @samsheff we can change the color, but not a feature I want to integrate just yet.  Moving this into a non-AR model would be especially useful when we support more than just the one badge tho.

Comment (samsheff) [inline app/controllers/badges_controller.rb]: Agreed. I'll go ahead and merge, but we should create an issue for it.

**Comment by aaronmallen** [comment]
@samsheff both of those things we can look at in the future, I think for now it's good as it is.  If you agree go lets merge it.

**Comment by samsheff** [comment]
Merging, @aaronmallen

**Pull request merged**